src/App.js

```diff
@@ -20,6 +20,7 @@ export default function App() {
   const dispatch = useDispatch();
 
   useEffect(() => {
+    document.documentElement.classList.add(`theme-light`);
```

Comment:
This initalized app with light mode enabled, needed while we don't store user preferences (ie settings endpoint)